src/app/views/users/_follow.html.erb

```diff
@@ -0,0 +1,6 @@
+<% if current_user && user != current_user %> 
+    <% follows = follows?(current_user.id, user.id) %>     
+    <%= link_to_if follows, 'Unfollow', user_followerships_path(user_id: user.id), class: 'button has-text', method: :delete do  %>
+       <%= link_to 'Follow', user_followerships_path(user_id: user.id), class: 'button has-text', method: :post  %>
```

Comment:
Is `.id` necessary? I think it will work fine without it and it'll be consistent with other views helpers

Reply:
thank you